conductor_api/src/config.rs

```diff
@@ -507,7 +507,16 @@ pub struct NetworkConfig {
     /// Global logging level output by N3H
     #[serde(default = "default_n3h_log_level")]
     pub n3h_log_level: String,
-    /// networking mode used by n3h
+    /// Overall mode n3h operates in.
+    /// Should be one of
+    /// * REAL
+    /// * MOCK
+    /// * HACK
+    /// REAL is the default and what should be used in all production cases.
+    /// MOCK is for using n3h only as a local hub that apps connect to directly, i.e. n3h will
+    /// not connect to any other n3h instance.
+    /// HACK is for trying out things and should only be used by n3h developers who know why
+    /// they need this (i.e. testing new code). Might get removed soon.
```

Comment:
@ddd-mtl, is this correct?
@Connoropolous, is this sufficient?

Reply:
Yes, except for HACK. I'd write:
```
/// HACK - Deprecated. Used by n3h developers only. Will get removed soon.
```